framework/kibocommerce/api/endpoints/cart/get-cart.ts

```diff
@@ -0,0 +1,41 @@
+import CookieHandler from '@framework/api/utils/cookie-handler'
```

Comment:
We should always do relative paths inside the providers folder, otherwise build will fail for other providers